composer.json

```diff
@@ -22,7 +22,8 @@
     "require-dev": {
         "doctrine/common": "~2.3",
         "symfony/property-access": "~2.2",
-        "phpunit/phpunit": "3.7.*"
```

Comment:
I didn't remove it.

But It should be removed IMO. But this is the place for another PR! ;-)